### PR TITLE
Syntax & Line Highlighting

### LINKED ISSUE #19: Editor Syntax Highlighting
**As a** program user
**I want** different elements of the code to be colored differently
**So that** I can more easily recognize which part of a program is which.

**Request Description**
Implement a way to recolor words in the program based on their recognition as a comment, instruction, register, immediate, or label.

### LINKED ISSUE #40: [BUG] Line numbers don't appear when the editor is empty
Line numbers do not appear when the app is open

1. Open the app
2. See that there are no line numbers

**Expected behavior**
Line numbers should appear, with only one line labeled with a 1.

### PR DESCRIPTION
**Pull request by TrevorBrunette**
Closes #19 
Closes #27 
Closes #40

### CODE REVIEW

File: src/main/java/com/ezasm/gui/ScrollBarUI.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+public class ScrollBarUI extends BasicScrollBarUI {
+
+    private final Theme theme;
+
+    public ScrollBarUI(Theme theme) {
+        this.theme = theme;
+    }
+
+    @Override
+    protected void configureScrollBarColors() {
```
Comment (TrevorBrunette) [inline src/main/java/com/ezasm/gui/ScrollBarUI.java:19]: Use background color for thumb?

**Review by TrevorBrunette — COMMENTED** [review]
LGTM

**Comment by TrevorBrunette** [comment]
Closes #54 
Closes #30 (should have been resolved earlier)
Closes #20

**Review by WyattRoss — APPROVED** [review]
LGTM! (Let's gamble, try merging)

**Pull request merged**